core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationDirective.js

```diff
@@ -267,28 +267,62 @@ oppia.directive('stateTranslation', [
               StateEditorService.setActiveStateName(
                 ExplorationInitStateNameService.displayed);
             }
-
-            var stateName = StateEditorService.getActiveStateName();
-
+            $scope.stateName = StateEditorService.getActiveStateName();
             $scope.stateContent = ExplorationStatesService
-              .getStateContentMemento(stateName);
+              .getStateContentMemento($scope.stateName);
             $scope.stateSolution = ExplorationStatesService
-              .getSolutionMemento(stateName);
+              .getSolutionMemento($scope.stateName);
             $scope.stateHints = ExplorationStatesService
-              .getHintsMemento(stateName);
+              .getHintsMemento($scope.stateName);
             $scope.stateAnswerGroups = ExplorationStatesService
-              .getInteractionAnswerGroupsMemento(stateName);
+              .getInteractionAnswerGroupsMemento($scope.stateName);
             $scope.stateDefaultOutcome = ExplorationStatesService
-              .getInteractionDefaultOutcomeMemento(stateName);
+              .getInteractionDefaultOutcomeMemento($scope.stateName);
             $scope.stateInteractionId = ExplorationStatesService
-              .getInteractionIdMemento(stateName);
+              .getInteractionIdMemento($scope.stateName);
             $scope.activeHintIndex = null;
             $scope.activeAnswerGroupIndex = null;
 
             var currentCustomizationArgs = ExplorationStatesService
-              .getInteractionCustomizationArgsMemento(stateName);
-            $scope.answerChoices = StateEditorService.getAnswerChoices(
-              $scope.stateInteractionId, currentCustomizationArgs);
+              .getInteractionCustomizationArgsMemento($scope.stateName);
+            var interactionId = $scope.stateInteractionId;
```

Comment:
is this an expected change?

Reply:
I were trying to resolve the conflicts in the branch then I have made them changes

Reply:
@dubeysandeep i seem that this is wrong changes beacause of conflicts in the branch. can you please tell me how to fix it?

Reply:
Have you followed the 5th bulletin of [this wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)?

Reply:
I have not followed this bulletin to resolve a merge conflicts.I have try to solve through the github then I have make these mistakes.

Reply:
Do not use GitHub as your editor. Always use the command line please. Otherwise you bypass the presubmit checks, which are important.

Reply:
sorry for this. can any one help me how to fix it.

Reply:
Thanks @DubeySandeep